commands.go

```diff
@@ -2068,6 +2068,16 @@ func (c *cmdable) ClientID() *IntCmd {
 	return cmd
 }
 
+func (c *cmdable) ClientUnblock(id int64, isError bool) *IntCmd {
```

Comment:
Can you please add 2 commands instead: `ClientUnblock` and `ClientUnblockWithError`?

Reply:
PR Updated